forms-flow-idm/keycloak/Dockerfile

```diff
@@ -1,5 +1,9 @@
-FROM jboss/keycloak
+FROM formsflow/forms-flow-idm-theme
 
-RUN sed -i -E "s/(<staticMaxAge>)2592000(<\/staticMaxAge>)/\1\-1\2/" /opt/jboss/keycloak/standalone/configuration/standalone.xml
-RUN sed -i -E "s/(<cacheThemes>)true(<\/cacheThemes>)/\1false\2/" /opt/jboss/keycloak/standalone/configuration/standalone.xml
-RUN sed -i -E "s/(<cacheTemplates>)true(<\/cacheTemplates>)/\1false\2/" /opt/jboss/keycloak/standalone/configuration/standalone.xml
\ No newline at end of file
+RUN mkdir -p /opt/keycloak/themes/formsflow
```

Comment:
@shuhaib-aot please update the keycloak path ith Docker file like /opt/bitnami/keycloak/themes/formsflow

Reply:
@amal-k-soman-aot  resolved please check